src/Components/ResourceLink.jsx

```diff
@@ -0,0 +1,25 @@
+export const ResourceLink = (props) => {
+  const { title, url, author, tags } = props;
+
+  return (
+    <div className="resource-link">
+      <h3>
+        <a href={url}>{title}</a>
```

Comment:
I prefer if the link opens in a new tab since users may be clicking on several of these links.